Meals/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class MealsConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
+    name = 'Meals'
```

Comment:
What is Meals about? How does that fit into https://github.com/KamilaWZ/alergy-detector/pull/9 requirements? I thought we name the app Allergy detector?